config/environments/development.rb

```diff
@@ -34,4 +34,16 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+      authentication: ENV['SMTP_SETTINGS_AUTHENTICATION'],
+      address: ENV['SMTP_SETTINGS_ADDRESS'],
+      port: ENV['SMTP_SETTINGS_PORT'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.